grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -2394,5 +2394,29 @@ class StockMovementService {
         }
         return lineItems
     }
+
+    def getDisabledMessage(StockMovement stockMovement, Location currentLocation, Boolean isEditing = false) {
```

Comment:
@jmiranda Perhaps `getMessageForDisabled` or `getMessageWhenDisabled` would be better?

Reply:
I think that Justin didn't mean the name of the method but just the way it's done.